paper.md

```diff
@@ -33,120 +33,183 @@ Shorter, but similar-spirit meetings have started in conjunction with conference
 
-- Balance between academics (say) and industry people?
 
-- Balance between scheduled program and unconference breakouts?
+## Measures of success
 
-## Participant selection
+- See notes on IRB above; maybe expand on that here.
 
-- THE most important part of the operation
+- How to evaluate a hack week?
 
-- importance of open methods
+- What are measures of success? How do we define success in the first place?
+
+
+## Conclusions and Discussion
+
+stuff here
 
-- axes of diversity
 
-- algorithmic selection
+# Appendix
 
-## Organizing a Hack Week
+While the main text of this paper remains fairly theoretical, we aim to lay out more practical advice regarding the organization of a hack week in the appendix of this document. 
 
 ### What to do before
 
-- WAY BEFORE: get IRB approval to track and study participants! This is a hard one, but very, very valuable.
+#### Obtaining an IRB to track and study participants
+For a large fraction of the hack weeks we have previously organized, studying the development of a hack week, its participants and outcomes was a stated goal of the workshop  **[GEO/NEURO: Is this true for you?]**. We included social scientists interested in scientific collaboration in the design process and organization, who both studied the workshop itself and gave valuable feedback about its outcomes. However, in order to be able to track and study participants, approval from Internal Review Boards is a must for workshops organized in the United States. This should be done as early as possible, and having assistance from researchers in the social sciences is invaluable, in particular for hack weeks in fields that generally don't involve human subject research. Ideally, participants should consent to be contacted at the application stage, which opens up the potential of tracking both participants and non-participants and allows for evaluation of the hack week. 
```

Comment:
Yes. We had folks from the local Data Science Studies group observing last year, and I hope they'll join in this year as well.